app/src/main/java/com/github/libretube/api/PlaylistsHelper.kt

```diff
@@ -167,18 +203,33 @@ object PlaylistsHelper {
         }.awaitAll()
     }
 
-    suspend fun exportPlaylists(): List<ImportPlaylist> = withContext(Dispatchers.IO) {
+    suspend fun exportPipedPlaylists(): List<PipedImportPlaylist> = withContext(Dispatchers.IO) {
         getPlaylists()
             .map { async { getPlaylist(it.id!!) } }
             .awaitAll()
             .map {
                 val videos = it.relatedStreams.map { item ->
                     "$YOUTUBE_FRONTEND_URL/watch?v=${item.url!!.toID()}"
```

Comment:
Question to reviewer: why force unwrap optional `url`?

Reply:
`url` is nullable and `toID()` only works with non-null strings. So we either need to use `!!` or `?`. Which one we use doesn't matter, because it should never become null. But asserting it as non-null is better in this case because `?` indicates it can be null.